ctx.go

```diff
@@ -1576,3 +1359,9 @@ func (c *DefaultCtx) IsFromLocal() bool {
 	}
 	return c.isLocalHost(ips[0])
 }
+
+func (c *DefaultCtx) Binding() *Bind {
+	return &Bind{
```

Comment:
is it ok to reinitialize the binder every time we call the bind method ? or should we put the instance in the ctx class ?

Reply:
I'll add it into the ctx